### PR TITLE
Feature lwdev 3979 partner id in sms

### PR DESCRIPTION
**Pull request by Darth-Jurassic**

### CODE REVIEW

File: src/Lykke.Job.Messages/QueueConsumers/SmsQueueConsumer.cs [hunk 1]
```diff
@@ -82,10 +84,14 @@ private void InitQueues()
                 nameof(HandleSmsRequestAsync),
                 $"SMS: Phone confirmation. Receiver: {request.PhoneNumber}, UTC: {DateTime.UtcNow}");
```
Comment (MikhailGerasimov) [inline src/Lykke.Job.Messages/QueueConsumers/SmsQueueConsumer.cs]: Should we hide personal data or at least obscure it?

Comment (Darth-Jurassic) [reply src/Lykke.Job.Messages/QueueConsumers/SmsQueueConsumer.cs]: I don't thank that this is a subject of current pull request, but we can escalate this question and fix it in future releases. Anyway phone number is logged now so we don't reveal any additional information by current update

File: src/Lykke.Job.Messages/QueueConsumers/SmsQueueConsumer.cs [hunk 2]
```diff
@@ -82,10 +84,14 @@ private void InitQueues()
                 nameof(HandleSmsRequestAsync),
                 $"SMS: Phone confirmation. Receiver: {request.PhoneNumber}, UTC: {DateTime.UtcNow}");
 
-            var msgText = await _smsTextGenerator.GenerateConfirmSmsText(request.MessageData.ConfirmationCode);
+            var msgText = await _templateFormatter.FormatAsync(nameof(SmsConfirmationTemplate), request.PartnerId, "EN",
```
Comment (tarurar) [inline src/Lykke.Job.Messages/QueueConsumers/SmsQueueConsumer.cs]: Wouldn't it be better to use const or enum instead of hard coded string?

Comment (Darth-Jurassic) [reply src/Lykke.Job.Messages/QueueConsumers/SmsQueueConsumer.cs]: https://lykkex.atlassian.net/browse/LWDEV-4615

**Pull request merged**